assignments/prototypes.js

```diff
@@ -92,19 +127,124 @@
-  // * Create two new objects, one a villain and one a hero and fight it out with methods!
\ No newline at end of file
+  // * Create two new objects, one a villain and one a hero and fight it out with methods!
+
+  function HeroVillan(attributes) {
+    Humanoid.call(this, attributes);
+      this.ability = attributes.ability;
+      this.finish = attributes.finish;
+  }
+  
+  HeroVillan.prototype = Object.create(Humanoid.prototype);
+  
+  HeroVillan.prototype.fight1 = function (target) {
+    if (target.healthPoints <= 0){
+    console.log(target.name + ' is looking hurt. Finish Him!')
+    return `${this.name} finishes ` + target.name + ` with ${this.finish}!!!`
+    }
+    else {
+    target["healthPoints"] = (target["healthPoints"] - 10)
+    return `${this.name} lashes out with the ${this.ability} ability.`;
+    }
+  };
+
+  HeroVillan.prototype.fight2 = function (target) {
+    if (target.healthPoints <= 0){
+    console.log(target.name + ' is looking hurt. Finish Him!')
+    return `${this.name} finishes ` + target.name + ` with ${this.finish}!!!`;
+    }
+    else {
+    target["healthPoints"] = (target["healthPoints"] - 15)
+    return `${this.name} swings his ${this.weapons} at ` + target.name;
+  }
+  };
+
+  const hero = new HeroVillan({
+    createdAt: new Date(),
+    dimensions: {
+      length: 2,
+      width: 2,
+      height: 3,
+    },
+    healthPoints: 50,
+    name: 'Jin Kazama',
+    team: 'Mishima Zaibatsu',
+    weapons: [
+      'Chain',
+    ],
+    language: [
+      'Common',
+      'Demonic',
+      'Japanese',],
+    ability: 'Lightning Screw Uppercut',
+    finish: 'Mishima Style 10-Hit Combo',
+  });
+
+  const villan = new HeroVillan({
+    createdAt: new Date(),
+    dimensions: {
+      length: 2,
+      width: 2,
+      height: 4,
+    },
+    healthPoints: 50,
+    name: 'Kazuya Mishima',
+    team: 'Mishima Zaibatsu',
+    weapons: [
+      'Fists',
+    ],
+    language: [
+      'Common',
+      'Demonic',
+      'Japanese',],
+    ability: 'Spinning Demon',
+    finish: 'Demonic Laser Beam',
+  });
+
+  // console.log(villan.fight1());
+  // console.log(hero.fight1());
+  // console.log(hero.fight2(villan));
+  // console.log(villan.fight2(hero));
+
+  // villan["healthPoints"] = -1;
+
+  // console.log(villan.healthPoints);
+  // console.log(villan);
+  
+  // console.log(hero.fight2(villan))
+
+  //HERO WIN EXAMPLE
+  // console.log('Round 1 Fight!')
+  // console.log(villan.fight1(hero));
+  // console.log(hero.fight1(villan));
+  // console.log(hero.fight2(villan));
+  // console.log(villan.fight2(hero));
+  // console.log(hero.fight1(villan));
+  // console.log(villan.fight2(hero));
+  // console.log(hero.fight2(villan));
+  // console.log(hero.fight2(villan));
+
+  //VILLAN WIN EXAMPLE
+  console.log('Round 2 Fight!')
+  console.log(hero.fight1(villan));
+  console.log(villan.fight1(hero));
```

Comment:
Great Job with the stretch!  I love how you added the ability to change someone's hp